packages/core/src/server/auth/token.ts

```diff
@@ -73,6 +74,27 @@ export class OAuth {
       }),
     });
 
+    if (response.status === 404) {
+      // Check for the deprecated authorize endpoint.
+      response = await fetch(`${wpUrl}/wp-json/wpac/v1/authorize`, {
+        headers: {
+          'Content-Type': 'application/json',
+          'x-wpe-headless-secret': apiClientSecret,
+        },
+        method: 'POST',
+        body: JSON.stringify({
+          code,
+          refreshToken: this.getRefreshToken(),
+        }),
+      });
+
+      if (response.status !== 404) {
+        log(
+          'The FaustWP WordPress plugin is out of date. Please update to the latest version.',
```

Comment:
I wonder if we should be more clear what is being deprecated that way users are not blindsided when we finally make the deprecation. 

Maybe something along the lines of "The `/wp-json/wpac/v1/authorize` endpoint will be deprecated soon. Please update to the latest version of the FaustWP plugin to be compatible."

Thoughts?

Reply:
I'm not sure how much detail we should give. I agree that a deprecation should usually warn you about what exactly is deprecated and how to fix it. However, I decided not to mention the endpoint in this case because it's not a public API. It's only consumed internally by the framework and isn't publicly documented. Our message isn't "we changed something that affects how you write your own code", but rather "we updated our own things, so you need to apply those updates to maintain compatibility." I think it's important for that distinction to be reflected in the message's wording.

Instead of mentioning specifics that could direct consumers attention towards their own code, we could just provide more detail on what features are affected. Maybe something like "Authentication and post previews will soon be incompatible with your version of the FaustWP plugin. Please update to the latest version."

Reply:
Great point. I think that's perfect